libs/coin-modules/coin-evm/src/bridge/js.ts

```diff
@@ -7,7 +7,7 @@ import {
 import { SignerContext } from "@ledgerhq/coin-framework/signer";
 import type { AccountBridge, Bridge, CurrencyBridge } from "@ledgerhq/types-live";
 import getAddressWrapper from "@ledgerhq/coin-framework/bridge/getAddressWrapper";
-import type { EvmAddress, EvmSignature, EvmSigner } from "../types/signer";
```

Comment:
shouldn't modification of coin-evm/*  be done in a separated PR? it's no longer a PR about tezos 🤔

Reply:
I wrote a general comment about this: "As a side effect to this PR, SignerContext has been simplified (to be more like the dedicated Bitcoin definition)".
I might separate it though...